trestle/core/control_io.py

```diff
@@ -211,12 +210,14 @@ def _insert_existing_text(self, part_label: str, existing_text: Dict[str, List[s
             for line in existing_text[part_label]:
                 self._md_file.new_line(line)
 
-    def _add_response(self, control: cat.Control, existing_text: Dict[str, List[str]]) -> None:
+    def _add_response(self, control: cat.Control, comp_dict: Dict[str, Dict[str, List[str]]]) -> None:
         """Add the response request text for all parts to the markdown along with the header."""
         self._md_file.new_hr()
         self._md_file.new_paragraph()
         self._md_file.new_header(level=2, title=f'{const.SSP_MD_IMPLEMENTATION_QUESTION}')
 
+        # need to add text per component
```

Comment:
I'm not sure you do here?

Reply:
or are you talking about documentation on how to setuo copmonents.

Reply:
I guess it was more a note to self.  I will remove it.  In the inner loop of that function it is adding text per component.